sphinx_gallery/gen_gallery.py

```diff
@@ -36,8 +36,8 @@
 from .directives import MiniGallery, ImageSg, imagesg_addnode
 
 
-_KNOWN_CSS = ('gallery', 'gallery-binder', 'gallery-dataframe',
-              'gallery-rendered-html')
+_KNOWN_CSS = ('sg_gallery', 'sg_gallery-binder', 'sg_gallery-dataframe',
+              'sg_gallery-rendered-html')
```

Comment:
Thanks @larsoner for thinking about this!

The change works perfectly fine with `nbsphinx`, since the `load_style` sub-module has been correctly updated.
The actual file name doesn't matter for my use case.